src/main/java/uk/gov/hmcts/darts/common/entity/SecurityGroupEntity.java

```diff
@@ -93,4 +93,8 @@ public class SecurityGroupEntity extends MandatoryCreatedModifiedBaseEntity {
     @Column(name = "display_name")
     private String displayName;
 
+    @NotAudited
+    @Column(name = "group_display_name")
+    private String legacyGroupDisplayName;
+
```

Comment:
Validation (See previous comment)?

Reply:
See earlier response to similar comment.